environments/pna/app-processes.yml

```diff
@@ -21,6 +21,8 @@ celery_processes:
       concurrency: 1
     async_restore_queue:
       concurrency: 1
+    celery:
+      concurrency: 1
```

Comment:
Is this not what line 9 accomplishes?
```
repeat_record_queue,celery,export_download_queue:
  concurrency: 1
```

Reply:
yeah, @dannyroberts pointed that out to me. Which leaves confused how this solved the problem, or how that problem existed before, but confident that this PR should not be needed.